reactfire/rollup.config.js

```diff
@@ -0,0 +1,25 @@
+import resolve from 'rollup-plugin-node-resolve';
+
+export default {
+    input: './pub/reactfire/index.js',
+    output: {
+      file: './pub/reactfire/bundle.js',
+      format: 'cjs',
+      name: 'reactfire'
+    },
+    external: [
+      'react',
+      'firebase/app',
+      'rxfire/auth',
+      'rxfire/database',
+      'rxfire/firestore',
+      'rxfire/storage',
+      'rxjs',
+      'rxjs/operators',
+    ],
+    plugins: [
+      resolve({
+        browser: true
```

Comment:
Looking [here](https://github.com/rollup/rollup-plugin-node-resolve#usage), it seems like `browser: true` is used when our package.json has a `browser` field. Our package.json doesn't have that - should we add it?

Reply:
We'd want to put a browser field in our package.json if we had a UMD to link to. I had to put in here to compile as one of our dependencies needed it. I can explore further as we work to support SSR and other environments.

Reply:
... actually I don't think I need this anymore, it was dep that was getting bundled in. Missed in externals.